pages/android/screencap.md

```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Take a screenshot of a mobile display.
+> This command can only be used through `adb shell`.
+> More infromation: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Take a screenshot:
+
+`adb shell screencap filename`
```

Comment:
```suggestion
`adb shell screencap {{path/to/file}}`
```

Use [double curly brackets](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) for placeholder path names, and since I assume you can place the file in any directory, use `path/to/file` instead.